kudos-services/src/main/java/org/exoplatform/kudos/model/Kudos.java

```diff
@@ -47,6 +47,8 @@ public class Kudos implements Serializable {
 
   private long              timeInSeconds;
 
+  private String            spacePrettyName;
```

Comment:
```suggestion
  @Exclude
  private String            spacePrettyName;
```
This information is transient, thus, you may not need it for `equals` & `hashcode`

Reply:
Done!